MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -67,7 +67,7 @@
         state = PlayerState.INIT;
         MuxStats.setHostDevice(new MuxDevice(ctx));
         MuxStats.setHostNetworkApi(new MuxNetworkRequests());
-        muxStats = new MuxStats(this, playerName, customerPlayerData, customerVideoData, sentryEnabled);
+        muxStats = new MuxStats(this, playerName, customerPlayerData, customerVideoData);
```

Comment:
Did you happen to pull the latest from master/merge from master? We added this after my branch was started, and we need to enable customers to turn off Sentry reporting. I can answer more questions about this tomorrow, but unless you added a way to do this elsewhere, we'll need to add this back in.